packages/discord.js-utilities/src/lib/PaginatedMessages/PaginatedMessage.ts

```diff
@@ -1202,14 +1202,14 @@ export class PaginatedMessage {
 	public addPageAction(action: PaginatedMessageAction, index: number): this {
 		if (index < 0 || index > this.pages.length - 1) throw new Error('Provided index is out of bounds');
 
-		if (!this.pageActions[index]) {
+		if (!this.pageActions.at(index)) {
 			this.pageActions[index] = new Map<string, PaginatedMessageAction>();
 		}
 
 		if (actionIsLinkButton(action)) {
-			this.pageActions[index]!.set(action.url, action);
+			this.pageActions.at(index)!.set(action.url, action);
```

Comment:
This change makes no sense to either be reported or changed...

Reply:
Fixed in [c2cc109](https://github.com/sapphiredev/utilities/pull/823/commits/c2cc1092a136fb360abf06e3b22e0dbed7415399)